doc/man1/openssl-fipsinstall.pod.in

```diff
@@ -216,6 +218,16 @@ This check is disabled by default.
 Configure the module to not allow short MAC outputs.
 See SP 800-185 8.4.2 and FIPS 140-3 ID C.D for details.
 
+=item B<-hmac_key_check>
+
+Configure the module to not allow small keys sizes when using HMAC.
+See SP 800-131Ar2 for details.
+
+=item B<-kmac_key_check>
+
+Configure the module to enable a run-time short key check when using KMAC.
```

Comment:
The wording for these two options ought to be the same apart from the `H` and `K`.